src/Pos/WorkMode.hs

```diff
@@ -149,8 +108,8 @@ type RawRealMode ssc =
     SscHolder ssc (
     NtpSlotting (
     SlottingHolder (
-    ContextHolder ssc (
     DBHolder (
+    ContextHolder ssc (
```

Comment:
Is it because `MonadDB` uses implicit `MonadReader`?

I would understand this argument if all these `MonadReader`s were from `mtl`, but it's not clear for readers from `ether`.